lib/MusicBrainz/Server/Edit/Role/AlwaysAutoEdit.pm

```diff
@@ -7,6 +7,7 @@ around 'edit_conditions' => sub {
     my ($orig, $self, @args) = @_;
 
     my $conditions = $self->$orig(@args);
+    $conditions->{always_auto_edit} = 1;
```

Comment:
I never liked the name of this package, since it's not always an auto-edit if you can still choose to make it votable. :)

Reply:
I didn't rename the package for now, but I did rename the condition at least.